ServerCore/Pages/Teams/List.cshtml

```diff
@@ -0,0 +1,50 @@
+﻿@page "/{eventId}/{eventRole}/Teams/List"
+@model ServerCore.Pages.Teams.ListModel
+
+@{
+    ViewData["Title"] = "List";
+}
+
+<h2>All Teams</h2>
+
+<p>
+    @if (Model.Event.IsTeamRegistrationActive || Model.EventRole == ModelBases.EventRole.admin)
+    {
+        <a asp-page="Create">Create New</a>
+    }
+    else
+    {
+        <div class="alert alert-danger" role="alert">
+            This event is not currently open for registration.
+        </div>
+    }
+</p>
+<table class="table">
+    <thead>
+        <tr>
+            <th>
+                @Html.DisplayNameFor(model => model.Teams[0].Name)
+            </th>
+            @if (Model.EventRole == ModelBases.EventRole.play && !Model.UserOnTeam && Model.Event.IsTeamMembershipChangeActive)
```

Comment:
Maybe a column for team size?

Reply:
Done. We'll have to keep an eye on perf for it (since getting a count involves querying all of the players in the event)